stable/redis-ha/templates/redis-haproxy-deployment.yaml

```diff
@@ -86,6 +86,10 @@ spec:
         - name: readonlyport
           containerPort: {{ default "6380" .Values.haproxy.readOnly.port }}
         {{- end }}
+        {{- if .Values.haproxy.metrics.enabled }}
+        - name: metrics-port
```

Comment:
I would allow this port name values to be configurable with https://github.com/helm/charts/pull/18881/files#diff-90d995e49ffa78848c1414023c10569bR115 here

You can keep the value that you put by default but make sure to update the README

Reply:
For use with a prometheus PodMonitors, or what for?

Reply:
I was thinking about separate PR for changing current behaviour of exporters configuration:
- custom labels
- allow custom port names for both services and deployments.

That's what @DandyDeveloper was talking about in #18166

Reply:
Yeah this make sense. Perfect!